templates/vue/src/components/NodeModal.vue

```diff
@@ -437,6 +537,7 @@ export default {
       formErrors: "",
       maxDescriptionLength: 250,
       addThumbnail: false,
+      tydeTypes: tydeTypes,
```

Comment:
This was not previously needed. Are you sure you need to add this?

Reply:
We need this because I use it in the template for checking tydeTypes. Previous it was only used in the script tag and this is imported from a different file.